src/plugins/mer/meremulatordevice.h

```diff
@@ -47,6 +47,7 @@ class MerEmulatorDeviceModel
     {}
 
     bool isNull() const { return d->name.isNull() || d->displayResolution.isNull() || d->displaySize.isNull(); }
+    bool isSdkProvided() const { return d->isSdkProvided; }
```

Comment:
In this commit we do not have the user defined models yet, so this change should be part of the following commit. OTOH the changes that make use of the MerDeviceModelComboBox inside MerEmulatorModeDialog are missing in this commit.

Reply:
Reworked.